src/platforms/apple/common/configuration/out-of-memory.mdx

```diff
@@ -1,24 +1,28 @@
 ---
 title: Out Of Memory
 sidebar_order: 10
-description: "Learn how to turn off Out Of Memory"
+description: "Learn how to turn off Out Of Memory tracking"
 ---
 
+<Note>
+This integration is only available before 8.0.0. We renamed it to <PlatformLink to="/configuration/watchdog-terminations/">watchdog terminations</PlatformLink> in the 8.0.0 release.
+</Note>
+
 This integration tracks out-of-memory (OOM) crashes based on heuristics. This feature is available for iOS, tvOS, and Mac Catalyst, works only if the application was in the foreground, and doesn't track OOMs for unit tests.
 
 When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, with details like the stack trace, tags, breadcrumbs, and so on, before the app terminates. With OOM crashes, the operating system can kill your app without further notice, which means the SDK can't write a report to disk.
 As a result, in the Apple SDK, we track OOM crashes during the app start based on heuristics, but getting the state of the app when an OOM occurs is challenging. To provide the same state as we do for a typical crash, we would periodically need to store the app state to disk. This would cause a lot of I/O, which in turn could negatively affect the application's performance. To avoid this, OOM events have less context than typical unhandled errors.
 
 When a user launches the app, the SDK checks the following statements and reports an OOM if all of them are true:
 
-* The app didn't crash on the previous run.
-* The app was in the foreground/active.
-* The user launched the app at least once. When your app produces an OOM during the first launch, the SDK can't detect it.
-* There was no debugger attached to the process of the app. If there was, our logic would falsely report OOMs whenever you restarted the app in development.
-* The app is not running on a simulator.
-* The OS did not update your app.
-* The user didn't update the OS of their device.
-* The user didn't terminate the app manually, and neither did the OS.
+- The app didn't crash on the previous run.
+- The app was in the foreground/active.
+- The user launched the app at least once. When your app produces an OOM during the first launch, the SDK can't detect it.
+- There was no debugger attached to the process of the app. If there was, our logic would falsely report OOMs whenever you restarted the app in development.
```

Comment:
```suggestion
- There was no debugger attached to the process of the app. (If there was, our logic would falsely report OOMs whenever you restarted the app in development.)
```

Reply:
Just not super clear why you committed the change for the other instance of this but not this one. It's pretty much the same language.

Reply:
Just missed it.